llmv_routes.py

```diff
@@ -52,11 +53,38 @@ def llm_verification_index():
     @authed_only
     def generate_for_challenge():
         """Add a route to CTFd for generating text from a prompt."""
+
+        import logging
+
+        class DebugConsoleHandler(logging.Handler):
+            def emit(self, record):
+                print(f"Custom Handler: {record.levelname} - {record.msg}")
+
+        log.addHandler(DebugConsoleHandler())
+
+        log.error("Initialized llm_route in verifications")
```

Comment:
Only part that really matters. Even with handlers, the right level, and no filters only error is being logged for some reason

![image](https://github.com/aivillage/llm_verification/assets/7213793/f2fe3e22-c9be-47fc-8922-d57e2f4040b9)